docs/contributor/assets/connectors.drawio.svg

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Do not edit this file with editors other than draw.io -->
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
I know this is a contributor doc and I certainly won't block merging because of that; but why don't we use the Kyma diagram guidelines for such diagrams?

Reply:
It is just easier to create the diagrams with the components available from draw.io. Next time, I will do the diagram with our Kyma components, but it is not worth it to redo this diagram as it is just a contributor doc